client/components/Overview/components/ProductInformation.jsx

```diff
@@ -1,45 +1,50 @@
-import LogClick from '../../LogClick';
-import facebook from '../../../../public/static/facebook.png';
-import twitter from '../../../../public/static/twitter.png';
-import pinterest from '../../../../public/static/pinterest.png';
+import '../styles/ProductInformationStyles.css';
+import RenderStars from '../../renderStars';
 
 //  function to render the general information to the DOM
 //  the price is rendered in a different way however
 //  if there is not a current price don't render anything
 //  if there is a current price but no sale price render the current price
 //  if there is botha current price and sale price render the sale price in red followed by the current price with a line through it
-const ProductInformation = ({ currentProduct, currentStyle }) => (
-  <div id="ProductInformation">
-    <div id="product-info-star-rating">Star Rating Placeholder</div>
-    <div id="product-info-category">Category: {currentProduct.category}</div>
-    <div id="product-info-title">Title: {currentProduct.name}</div>
+const ProductInformation = ({
+  currentProduct, currentStyle, overviewAverage, reviewScroll,
+}) => (
+  <div id="productinformation-container">
+    <div id="product-info-star-rating">
+      <div id="overviewAverage">
+        <RenderStars rating={overviewAverage} />
+      </div>
+      <div
+        id="review-text"
+        onClick={
+        () => reviewScroll.current.scrollIntoView({ behavior: 'smooth' })
+      }
+      >read all reviews
+      </div>
+    </div>
+    <div id="product-info-category">{currentProduct.category}</div>
+    <div id="product-info-title">{currentProduct.name}</div>
 
     {currentStyle ? currentStyle.sale_price
       ? (
-        <div style={{ display: 'inline-block' }}>
+        <div id="productinformation-sale" style={{ display: 'inline-block' }}>
           <div id="product-info-price-discount" style={{ color: 'red', display: 'inline-block' }}>
-            {currentStyle.sale_price}
+            ${currentStyle.sale_price}
           </div>
           <div id="product-info-price-line" style={{ textDecoration: 'line-through', display: 'inline-block' }}>
-            {currentProduct.default_price}
+            ${currentProduct.default_price}
           </div>
         </div>
       )
-      : <div id="product-info-price">Price: {currentProduct.default_price}</div>
-      : <div id="product-info-price">Price: {currentProduct.default_price}</div>}
-
-    <div id="product-info-overview">Overview: {currentProduct.description}</div>
-    <div id="product-info-share">
-      <img id="product-info-share-facebook" src={facebook} alt="" onClick={() => LogClick('img', 'Overview')} />
-      <img id="product-info-share-twitter" src={twitter} alt="" onClick={() => LogClick('img', 'Overview')} />
-      <img id="product-info-share-pinterest" src={pinterest} alt="" onClick={() => LogClick('img', 'Overview')} />
-    </div>
+      : <div id="product-info-price">${currentProduct.default_price}</div>
```

Comment:
is DevTools giving you any grief for having non-unique ids?

Reply:
no, this was because the images are different sizes and allows me to manipulate them certain ways.